### PR TITLE
feat!: update transparency (MAPCO-2764)

### PR DESCRIPTION
**Pull request by asafMasa**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -17,6 +17,9 @@
       "skipFiles": ["${workspaceFolder}/node_modules/**/*.js", "<node_internals>/**/*.js"],
       "outFiles": ["${workspaceFolder}/dist/**/*.js", "!**/node_modules/**"],
       "program": "${workspaceFolder}/src/index.ts",
+      // "args": [
```
Comment (ronenkapelian) [inline .vscode/launch.json:20]: why its commented?

File: src/publishCommand/publishManager.ts [hunk 2]
```diff
@@ -122,11 +123,13 @@ export class PublishManager {
...
         name: layerName,
         tilesPath: row.tilesPath,
-        format: row.format as TileFormat,
+        format: row.format as TileOutputFormat,
       });
     } catch (exception) {
       this.logger.error(row, 'Failed to handle row for data:');
       throw exception;
+    } finally {
+      this.logger.info(row, 'Finished to handle row');
```
Comment (ronenkapelian) [inline src/publishCommand/publishManager.ts:132]: maybe add some debug log that presents the object data that was sent? sometimes it helps to understand.
not sure - asking and consulting on the flight :)

Comment (asafMasa) [reply src/publishCommand/publishManager.ts:132]: https://mapcolonies.atlassian.net/browse/MAPCO-2803 added

File: .vscode/launch.json (hunk 1 above)
Comment (asafMasa) [reply .vscode/launch.json:20]: fixed

File: src/publishCommand/publishManager.ts [hunk 3]
```diff
@@ -254,8 +259,16 @@ export class PublishManager {
       this.logger.error('invalid data, missing required field: format');
       throw new Error('invalid data');
     }
-    if (row.format !== TileFormat.PNG && row.format !== TileFormat.JPEG) {
-      this.logger.error(`invalid data, only acceptable values: ${TileFormat.PNG} or ${TileFormat.JPEG}`);
+    if (row.format !== TileOutputFormat.PNG && row.format !== TileOutputFormat.JPEG) {
+      this.logger.error(`invalid data, 'format' only acceptable values: ${TileOutputFormat.PNG} or ${TileOutputFormat.JPEG}`);
+      throw new Error('invalid data');
+    }
+    if (row.transparency === '') {
```
[the remark this answers is not in the document]
Comment (asafMasa) [reply src/publishCommand/publishManager.ts:266]: legacy code convention

**Review by ronenkapelian — APPROVED** [review]

**Pull request merged**